components/expander.jsx

```diff
@@ -32,7 +32,7 @@ var Expander = React.createClass({
     return (
       <div className="expander-container">
         <div className={className} tabIndex="0" onBlur={this.collapse} onFocus={this.expand}>
-          <div onMouseDown={this.onClick} className="expander-header">
+          <div onClick={this.handleClick} className="expander-header">
```

Comment:
Hmm, good question, and I'm not sure what the answer is, though the MDN page on [using the button role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_button_role) seems helpful. Perhaps it might be better to focus on the tests for this PR, and file an accessibility bug that we can address after landing this?